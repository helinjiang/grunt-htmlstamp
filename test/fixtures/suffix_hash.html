<script type="text/javascript" src="./test1.js"></script>
<script type="text/javascript" src="./test2.js"></script>
<script type="text/javascript"></script>