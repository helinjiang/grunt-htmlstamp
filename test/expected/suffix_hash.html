<script type="text/javascript" src="./test1.js?_v=241f131860"></script>
<script type="text/javascript" src="./test2.js"></script>
<script type="text/javascript"></script>